frontend/app/configuration/Config.scala

```diff
@@ -47,8 +49,13 @@ object Config {
   def idWebAppSigninUrl(uri: String): String =
     (idWebAppUrl / "signin") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & idMember
 
-  def idWebAppRegisterUrl(uri: String): String =
+  def idWebAppRegisterUrl(uri: String, abTestVariant: CheckoutFlowVariant = A): String ={
+
+    val idMember = "clientId" -> abTestVariant.identitySkin
+
     (idWebAppUrl / "register") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & idMember
```

Comment:
Note that `idMember` is hiding the wider-scoped `idMember` variable here. Perhaps better to inline this variable, or use a different name?